test/config/bundles.js

```diff
@@ -0,0 +1,38 @@
+import chai from 'chai';
```

Comment:
I personally don’t think chai is necessary for testing, and Node.js assert library is more than enough.